README.md

```diff
@@ -51,3 +51,31 @@ Arangopipe represents metadata as a graph. The nodes of the graph above are the
 20. (Edge) Project - Models: This captures the models associated with a project.
 
 The data associated with the nodes and edges of the graph are stored as documents. The documents do not have a fixed structure and represent data using key-value pairs. This offers flexibility and permits users to decide the metadata elements that they would like to store. This also permits users to store metadata from any machine learning tech stack in Arangopipe.
+
+## Installing Arangopipe
+
+This repository contains **Arangopipe** and examples to illustrate how it can be used with machine learning tools like **hyperopt** and **MLFlow**. To install **Arangopipe**, do the following:
+
+1.  Install ArangoDB
+
+    `docker run -p 8529:8529 -e ARANGO_ROOT_PASSWORD=openSesame arangodb/arangodb:3.4.6`
+    
+2.  Install python-arango
```

Comment:
2. Install prerequisites

`pip install python-arango PyYaml`

and then we can drop 3.

Reply:
also we should probably fix the versions here...
Maybe use a `pip install -r requirements.txt` ?

Reply:
Agree.